core/client.py

```
@@ -129,19 +242,29 @@ def _init_client(self):
                 cached_client = cache[session_path]
                 if cached_client and cached_client.is_connected():
-                    logger.debug(f"[Client Cache] 复用现有的 Telegram 客户端连接: {session_path}")
+                    logger.debug(
+                        f"[Client Cache] 复用现有的 Telegram 客户端连接: {session_path}"
+                    )
                     self.client = cached_client
                     return
                 else:
-                    logger.debug(f"[Client Cache] 缓存的客户端已断开，正在重新创建: {session_path}")
+                    logger.debug(
+                        f"[Client Cache] 缓存的客户端已断开，正在重新创建: {session_path}"
```

Ruff (0.15.12)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)